tests/e2e/taqueria-plugin-taquito-flextesa.spec.ts

```diff
@@ -0,0 +1,138 @@
+import fsPromises from "fs/promises"
+import { generateTestProject, getContainerName } from "./utils/utils";
+import { exec as exec1 } from "child_process";
+import utils from 'util'
+const exec = utils.promisify(exec1)
+
+const taqueriaProjectPath = 'e2e/auto-test-taquito-flextesa-plugin';
+const contractRegex = new RegExp(/(KT1)+\w{33}?/);
+let environment: string;
+let dockerName: string = "local"
+
+describe("E2E Testing for taqueria taquito plugin",  () => {
+
+    beforeAll(async () => {
+        await generateTestProject(taqueriaProjectPath, ["taquito", "flextesa"]);
+        await exec(`cp e2e/data/config-taquito-flextesa-local-sandbox-test-environment.json ${taqueriaProjectPath}/.taq/config.json`);
+        await exec(`cp e2e/data/hello-tacos.tz ${taqueriaProjectPath}/artifacts/`)
+        await exec(`cp e2e/data/increment.tz ${taqueriaProjectPath}/artifacts/`)
+        await exec(`taq stop sandbox ${dockerName}`, {cwd: `./${taqueriaProjectPath}`})
```

Comment:
I don't see any reason why we should be stopping a sandbox in the beforeAll section

Reply:
If you run the tests locally, its quite possible that you'll have a local sandbox running. As such, `taq start sandbox` would fail.